pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -119,6 +119,11 @@ spec:
             - "--drivername={{ .DriverNamePrefix }}cephfs.csi.ceph.com"
             - "--pidlimit=-1"
             - "--forcecephkernelclient={{ .ForceCephFSKernelClient }}"
+            {{ if .CSILogRotation }}
+            - "--logtostderr=false"
+            - "--alsologtostderr=true"
+            - "--log_file={{ .CsiLogDirPath }}/{{ .DriverNamePrefix }}cephfs.csi.ceph.com/cephcsi/log/cephfs-provisioner/csi-cephfsplugin.log"
```

Comment:
`{{ .CsiLogDirPath }}/{{ .DriverNamePrefix }}cephfs.csi.ceph.com/cephcsi/log/cephfs-provisioner/csi-cephfsplugin.log` to `{{ .CsiLogDirPath }}/{{ .DriverNamePrefix }}cephfs.csi.ceph.com/log/controller-plugin/csi-cephfsplugin.log`

@parth-gr how about the above, we dont need to add `cephcsi` and as you already know we are planning to switch from provisioner to controller-plugin in the new operator for the naming convention to match the CSI specifications.

Reply:
If you agree please take care of this in all other places as well.